app/services/ipam/base.rb

```diff
@@ -1,23 +1,40 @@
 module IPAM
   class Base
+    extend ActiveModel::Naming
```

Comment:
They are required for Active Model Errors, see http://api.rubyonrails.org/classes/ActiveModel/Errors.html